Update The Project With Goodies (#31)
ionixjunior/Xalendar: README.md

# Xalendar [![codecov](https://codecov.io/gh/ionixjunior/Xalendar/branch/master/graph/badge.svg)](https://codecov.io/gh/ionixjunior/Xalendar)

> ![xalander logo](./assets/logo.png)
> 
> Xalender is a calendar view for Xamarin.Forms applications.

Author: ```suggestion
![Xalendar logo](./assets/logo.png)

Xalendar is a calendar view for Xamarin.Forms applications.
```

![screenshot](./assets/screenshot.png)

## Getting Started

> Install Xalander using NuGet with the following command line to your shared library project.

Author: ```suggestion
Install Xalendar using NuGet with the following command line to your shared library project.
```

```console

> > dotnet add package Xalender

Author: ```suggestion
> dotnet add package Xalendar
```

```

> To use the Xalender component, you'll need to add the following namespace to the top of your XAML views.

Author: ```suggestion
To use the Xalendar component, you'll need to add the following namespace to the top of your XAML views.
```
